homeassistant/block_async_io.py

```diff
@@ -153,12 +153,12 @@ class BlockedCalls:
     calls: set[BlockingCall]
 
 
-_BLOCKED_CALLS = BlockedCalls(set())
+BLOCKED_CALLS = BlockedCalls(set())
```

Comment:
I think it would be OK to keep this private to signal it should not be accessed by anyone (except tests)

Reply:
dc2cb9a763c